test/test-storage.ts

```diff
@@ -88,3 +88,24 @@ test("retrieving mulitple files with .fetchAllFiles()", async (t) => {
     return a.name > b.name ? 1 : -1;
   }
 });
+
+test("update the bcp47 tag to database", async (t) => {
+  const { db, storage } = t.context;
+  // At first, nothing in the DB
+  t.is(await db.packageInfo.count(), 0);
+  await storage.updateBCP47Tag("en");
+  // Now there's one package info record in the DB!
+  t.is(await db.packageInfo.count(), 1);
+});
+
+test("retrieve bcp47tag from the database", async (t) => {
+  const { storage } = t.context;
+  await storage.updateBCP47Tag("en");
+  const maybePackageInfo = await storage.fetchPackageInfo();
+  if (maybePackageInfo == undefined) {
+    t.fail("packageInfo is undefined.");
+  } else {
+    const bcp47Tag = maybePackageInfo.bcp47Tag;
+    t.is(bcp47Tag, "en");
+  }
```

Comment:
If maybePackageInfo is not surrounded by else{}, an error will be reported.

Reply:
Ah, my bad! I thought `t.fail()` had [the `never` type](https://www.typescriptlang.org/docs/handbook/basic-types.html#never), but [it does not](https://github.com/avajs/ava/blob/b71f396df7b9d74ef24262b6840df1593cad3798/index.d.ts#L141). Kinda annoying :/

The if/else will have to do!